iroha_crypto/src/varint.rs

```diff
@@ -0,0 +1,134 @@
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Debug)]
+pub struct VarUint {
```

Comment:
Please add module docs with a link to the spec https://github.com/multiformats/unsigned-varint